.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,49 @@
+---
+name: rsa-keygen Issue
+about: 'Template for submitting an issue'
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Pre-requisites
+
+Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**
+
+- [ ] I am running the latest version
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
+
+# Expected Behavior
+
+(Please describe the behavior you are expecting)
+
+# Current Behavior
+
+(What is the current behavior?)
+
+# Bug Details
+
+(Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.)
+
+## Steps to Reproduce
+
+(Please provide detailed steps for reproducing the issue.)
+
+1. 
+2. 
```

Comment:
I would prefer to rely on the github auto numbering i.e.
```
1.
1.
1.
```
Rather than hard-code the numbers